content/en/docs/concepts/storage/volume-snapshots.md

```diff
@@ -22,61 +23,62 @@ This document describes the current state of `VolumeSnapshots` in Kubernetes. Fa
 [volume snapshot class](/docs/concepts/storage/volume-snapshot-classes/) and
 the administrator must have created and configured that class in order for dynamic
 provisioning to occur.
 
 ### Binding
 
-A user creates, or has already created in the case of dynamic provisioning, a `VolumeSnapshot` with a specific amount of storage requested and with certain access modes. A control loop watches for new VolumeSnapshots, finds a matching VolumeSnapshotContent (if possible), and binds them together. If a VolumeSnapshotContent was dynamically provisioned for a new VolumeSnapshot, the loop will always bind that VolumeSnapshotContent to the VolumeSnapshot. Once bound, `VolumeSnapshot` binds are exclusive, regardless of how they were bound. A VolumeSnapshot to VolumeSnapshotContent binding is a one-to-one mapping.
+A user creates, or has already created in the case of dynamic provisioning, a `VolumeSnapshot` with a specific set of parameters. A control loop watches for new `VolumeSnapshot` requests, finds a matching `VolumeSnapshotContent` (if possible), and binds them together. If a `VolumeSnapshotContent` was dynamically provisioned for the new `VolumeSnapshot`, the loop will always bind that `VolumeSnapshotContent` to the `VolumeSnapshot`. Once bound, `VolumeSnapshot` binds are exclusive, regardless of how they were bound. A `VolumeSnapshot` to `VolumeSnapshotContent` binding is a one-to-one mapping.
 
-VolumeSnapshots will remain unbound indefinitely if a matching VolumeSnapshotContent does not exist. VolumeSnapshots will be bound as matching VolumeSnapshotContents become available.
+*Volume snapshots* will remain unbound indefinitely if a matching `VolumeSnapshotContent` does not exist. *Volume snapshots* will be bound as matching `VolumeSnapshotContent` resources become available.
 
 ### Persistent Volume Claim in Use Protection
 
-The purpose of the Persistent Volume Claim Object in Use Protection feature is to ensure that in-use PVC API objects are not removed from the system (as this may result in data loss).
+The purpose of the *Persistent Volume Claim Object in Use* Protection feature is to ensure that in-use PVC API objects are not removed from the system (as this may result in data loss).
 
 If a PVC is in active use by a snapshot as a source to create the snapshot, the PVC is in-use. If a user deletes a PVC API object in active use as a snapshot source, the PVC object is not removed immediately. Instead, removal of the PVC object is postponed until the PVC is no longer actively used by any snapshots. A PVC is no longer used as a snapshot source when `ReadyToUse` of the snapshot `Status` becomes `true`.
 
 ### Delete
 
-Deletion removes both the `VolumeSnapshotContent` object from the Kubernetes API, as well as the associated storage asset in the external infrastructure.
+Deletion removes the `VolumeSnapshotContent` object from both the Kubernetes API, as well as the associated storage asset in the external infrastructure.
```

Comment:
Clarify here that deletion is triggered by deleting the `VolumeSnapshot` object, not by deleting the `VolumeSnapshotContent` object.

Reply:
I would like to leave this to others who have more knowledge about the delete behavior, especially since some changes are being proposed in this area.

Reply:
@xing-yang can you clarify here how you would like this to be worded? I think it's important to clarify that deletion is triggered by deleting the VolumeSnapshot object, not the Content object.